core/src/main/java/org/infinispan/configuration/parsing/Parser.java

```diff
@@ -1870,33 +1865,22 @@ protected void parseTransaction(XMLExtendedStreamReader reader, ConfigurationBui
       ParseUtils.requireNoContent(reader);
    }
 
-   protected void parseEviction(XMLExtendedStreamReader reader, ConfigurationBuilder builder) throws XMLStreamException {
-      log.evictionDeprecated();
+   private void parseEviction(XMLExtendedStreamReader reader, ConfigurationBuilder builder) throws XMLStreamException {
+      log.elementDeprecatedUseOther(Element.EVICTION, Element.MEMORY);
       for (int i = 0; i < reader.getAttributeCount(); i++) {
          String value = replaceProperties(reader.getAttributeValue(i));
          Attribute attribute = Attribute.forName(reader.getAttributeLocalName(i));
          switch (attribute) {
-            case STRATEGY: {
-               builder.eviction().strategy(EvictionStrategy.valueOf(value));
+            case STRATEGY:
+            case THREAD_POLICY:
+            case TYPE:
+               log.ignoreXmlElement(attribute);
                break;
-            }
-            case MAX_ENTRIES: {
+            case MAX_ENTRIES:
                log.evictionMaxEntriesDeprecated();
```

Comment:
We can remove this now too since all of eviction is deprecated.

Reply:
@wburns Hmm, this has got me thinking. Can we make these changes to the parser for 9.1.x, or do we need to hold off this for 9.2.x?